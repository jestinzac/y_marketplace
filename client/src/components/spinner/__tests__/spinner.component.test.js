import { render, screen } from "@testing-library/react";

import Spinner from "../spinner.component";

describe("Spinner", () => {
  it("should render the component", () => {
    const { asFragment } = render(<Spinner />);

    expect(screen.getByTestId("loader")).toHaveClass("animate-pulse");
    expect(screen.getByTestId("loader")).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
