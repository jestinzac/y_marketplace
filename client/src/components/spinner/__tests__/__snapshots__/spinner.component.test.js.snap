// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render the component 1`] = `
<DocumentFragment>
  <div
    class="grid grid-cols-5 gap-3 animate-pulse"
    data-testid="loader"
  >
    <div
      class="bg-gray-200 grid content-start"
    />
    <div
      class="bg-gray-200 col-span-4 jobs-info-container h-screen overflow-scroll scroll-smooth"
    />
  </div>
</DocumentFragment>
`;
